client/src/components/shared/Tracker.js

```diff
@@ -1,14 +1,33 @@
+import React, { useContext } from "react";
+import { Link } from "react-router-dom";
+import MoodForm from "../forms/MoodForm";
+import { AuthContext } from "../../providers/AuthContext";
 
-import React from 'react';
-import Login from '../auth/Login';
 
 const Tracker = () => {
-  
-  return (
-    <Login/>
-  )
-}
- 
+  const auth = useContext(AuthContext);
+  const {
+    data: { user }
+  } = auth;
 
+  const loggedIn = () => {
```

Comment:
You need not this function, remember ProtectedRoute will handle this logic.
This component could be `const Tracker = () => <MoodForm />;` or you could update your ProtectedRoute with path='/' to have component={MoodForm} but I'm not sure what your plans on for Tracker and it may have more than just <MoodForm> in which case my first suggestion stands

Reply:
Oh yeah...I'm not sure either what my plans are. YOur right i'll remove the function and just have it return <MoodForm/> and keep the routing the same. I can always remove Tracker if it's not needed later.